exercises/simple-cipher/src/test/java/SubstitutionCipherTest.java

```diff
@@ -0,0 +1,94 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+
+/**
+ * Step 2: Substitution cipher
+ */
+public class SubstitutionCipherTest {
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+    
+    private Cipher cipherWithDefaultKey;
+    private static final String key = "abcdefghij";
+
+    @Before
+    public void setup() {
+        cipherWithDefaultKey = new Cipher(key);
+    }
+    
+    /**
+     * Can encode
+     */
+    @Ignore("Remove to run test")
+    @Test
+    public void cipherCanEncode() {
+        assertEquals("abcdefghij", cipherWithDefaultKey.encode("aaaaaaaaaa"));
+    }
+    
+    /**
+     * Can decode
+     */
+    @Ignore("Remove to run test")
+    @Test
+    public void cipherCanDecode() {
+        assertEquals("aaaaaaaaaa", cipherWithDefaultKey.decode("abcdefghij"));
+    }
+    
+    /**
+     * Is reversible. I.e., if you apply decode in a encoded result, you must see
+     * the same plaintext encode parameter as a result of the decode method
+     */
+    @Ignore("Remove to run test")
+    @Test
+    public void cipherIsReversibleGivenKey() {
+        String plainText = "abcdefghij";
+        assertEquals(plainText, cipherWithDefaultKey.decode(cipherWithDefaultKey.encode("abcdefghij")));
+    }
+    
+    /**
+     * Can double shift encode
+     */
+    @Ignore("Remove to run test")
+    @Test
+    public void cipherCanDoubleShiftEncode() {
+        String plainText = "iamapandabear";
+        String cipherText = "qayaeaagaciai";
```

Comment:
I'd prefer to be consistent with having these variables; either have `plainText` and `cipherText` variables for each test or for none of them, if that makes sense? 🙂